samples/wifi/sr_coex/test/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
Please move all tests to the `tests` directory, samples are for demonstration not for testing, see https://docs.zephyrproject.org/latest/samples/sample_definition_and_criteria.html#samples-must-not-be-used-as-a-testcase.

And then convert each `overlay-wifi*` to a test case, I would also recommend removing the test code from the sample and convert from compile time tests to run time tests (no changes to the configuration).